amundsen_application/static/js/components/Feedback/FeedbackForm/styles.scss

```diff
@@ -52,8 +52,13 @@ input[type="radio"] {
   margin: 5px;
```

Comment:
Because of how this margin is actually used, I'd rather go smaller than larger if we had to change it. 8px widens the buttons more than necessary:
![image](https://user-images.githubusercontent.com/1790900/57256862-a748df00-700c-11e9-859f-4a060336488d.png)

Even so, 5px was chosen as a hack to align the labels with the radio button inputs. Any less than 5px and you'll notice the radio buttons become even more offset from the buttons. If we want to enforce the grid here, I think the proper thing to do would be to re-do the markup for this form as a separate task to be less "hacky", because adhoc changes will mess up the hacked parts of the layout.

Reply:
Oh right, I forgot about this. Probably best to leave this alone then.